src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Constants/RequestTracingConstants.cs

```diff
@@ -35,7 +35,10 @@ internal class RequestTracingConstants
         public const string FeatureFlagMaxVariantsKey = "MaxVariants";
         public const string FeatureFlagUsesVariantConfigurationReferenceTag = "ConfigRef";
         public const string FeatureFlagFeaturesKey = "FFFeatures";
-        public const string SignalRUsedTag = "UsesSignalR";
+        public const string FeaturesKey = "Features";
+        public const string LoadBalancingEnabledTag = "LB";
+        public const string SignalRUsedTag = "SignalR";
+        public const string FailoverRequestTag = "FailoverRequest";
```

Comment:
To keep a check on the size of Correlation-Context header, can we shorten the tag to "Failover"?